src/coreclr/pal/src/misc/perfjitdump.cpp

```diff
@@ -61,6 +61,22 @@ namespace
         JIT_CODE_LOAD = 0,
     };
 
+#if defined(__linux__)
+    static uint32_t sys_gettid()
+    {
+        return syscall(SYS_gettid);
+    }
+#else
+    static uint32_t sys_gettid()
```

Comment:
What is this used for? Should this use `PlatformGetCurrentThreadId` instead of rolling it own clone of this logic?